demos/storybook/stories/selection-toolbar/with-basic-usage.stories.ts

```diff
@@ -0,0 +1,23 @@
+import { text } from '@storybook/addon-knobs';
+
+export const withBasicUsage = (): any => ({
+    template: `
+       <pxb-selection-toolbar [title]="title" [subtitle]="state.selected || subtitle">
+        <ng-container pxb-menu>
+            <button mat-menu-item (click)="updateSubtitle('Test Item 1', state)">Test Item 1</button>
+            <button mat-menu-item (click)="updateSubtitle('Test Item 2', state)">Test Item 2</button>
+            <button mat-menu-item (click)="updateSubtitle('Test Item 3', state)">Test Item 3</button>
+        </ng-container>
+       </pxb-selection-toolbar>
+    `,
+    props: {
+        title: text('title', 'Title'),
+        subtitle: text('subtitle', 'Subtitle'),
+        updateSubtitle: (str: string, state): void => {
+            state.selected = str;
```

Comment:
It feels like the storybook gets out of sync...when you select an item from the dropdown, the toolbar is updated, but the knob still has the old label (across all stories)